app/src/main/java/net/bible/android/view/activity/page/screen/DocumentWebViewBuilder.kt

```diff
@@ -327,7 +327,7 @@ class DocumentWebViewBuilder @Inject constructor(
         toggleWindowButtonVisibility(true)
         timerTask?.cancel()
 
-        if(mainBibleActivity.fullScreen) {
+        if(true || mainBibleActivity.fullScreen) {
```

Comment:
Something temporary? true will always pass as true :)

Reply:
Temporary indeed :-) Good catch, forgot this that way. I'll remove if block.